### ISSUE TITLE
DXM + Ketamine: Low Risk & No Synergy -> Low Risk & Synergy

### ISSUE THREAD

**Issue by konrad-komisarczyk** [TripSit/drugs#28]
I think we should modify the combination according to the title.

Both substances exert their effects mainly via NMDA antagonism and there is no data on potential mechanisms causing decrease of effects, which implies synergy.

**Comment by LunaUrsa** [comment]
Hey there! Thanks for this, do you happen to have any online sources? @utaninja Do you agree with this?

**Comment by konrad-komisarczyk** [comment]
On DXM and Ketamine pharmacology i can source wikipedia pages:
* [https://en.wikipedia.org/wiki/Dextromethorphan](https://en.wikipedia.org/wiki/Dextromethorphan)
* [https://en.wikipedia.org/wiki/Ketamine](https://en.wikipedia.org/wiki/Ketamine)

The receptor affinities tables have references that I belive are legit.

**Comment by utaninja** [comment]
> On DXM and Ketamine pharmacology i can source wikipedia pages:
> 
> * https://en.wikipedia.org/wiki/Dextromethorphan
> * https://en.wikipedia.org/wiki/Ketamine
> 
> The receptor affinities tables have references that I belive are legit.

I agree with this. This is a very logical induction as far as I know and it seems everything supports this.

**Comment by utaninja** [comment]
From discord user RedDefault-
"Ketamine+DXM = Low to no SS concerns (NMDA toxicity may be of a concern, SS concerns can be compared to Ket+SSRI, if NMDA toxicity is not an issue then it should be synergy+low risk, but that will have to be looked at)"
-
I'm not familiar with NMDA toxicity but will try to take a look when I can. If anyone has any input on this please feel free to share :)
